docs/changelog.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 =========
 
 * :feature:`5816` The NFT images will not be automatically rendered now. It is made so to prevent a known security issue, that may result in leakage of your privacy (read https://medium.com/@alxlpsc/critical-privacy-vulnerability-getting-exposed-by-metamask-693c63c2ce94 ). You can add domains you trust to the whitelisted domain in the NFT setting.
+* :feature:`5625` Global address book will now receive automatic updates regularly i.e. known addresses will be associated with names.
```

Comment:
Perhaps let's not advertise this. As discussed this will only have some of our small updates to our own remote data and not the usage of other sources. So such a changelog probably may create expectations